docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,88 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there were three configuration settings for how Mobile UCRs were restored and sent to devices:
+
+- Version 1.0: Bundled all reports into a single fixture during the restore process.
+- Version 1.5: Sent both V1 and V2 restore formats.
+- Version 2.0: Split UCRs into multiple fixtures within the restore.
```

Comment:
what do you mean by multiple fixtures?

Reply:
One of the changes from v1->v2 is that all reports are no longer stored in a single shared fixture anymore. Instead, each report has its own fixture which gets synced with the mobile device. More information can be found in the migration documentation [here](https://dimagi.atlassian.net/wiki/spaces/GS/pages/2146604299/Moving+to+Mobile+UCR+V2).